root/statistics/stats.js

```diff
@@ -85,6 +85,12 @@ const stats = {
     description: l('Artists with gender set to male'),
     label: l('Male Artists'),
   },
+  'count.artist.gender.not_applicable': {
+    category: 'artist-other',
+    color: '#ff0000',
+    description: l('Artists with gender set to not applicable'),
+    label: l('Artists with gender set to not applicable'),
```

Comment:
Hmm. I guess it could. I looked at "Male Artists" (eww, that case) and I thought I should fit "Artists" there somehow, but it's actually very hard to say this shortly, so maybe Gender Not Applicable is indeed the best option.

Reply:
Yeah, I almost suggested "Not-applicable-gender Artists" before realizing how dumb and forced that sounded.